camayoc/tests/qpc/ui/conftest.py

```diff
@@ -20,8 +22,39 @@ def pytest_collection_modifyitems(config, items):
                 item.add_marker(skip_ui)
 
 
+@pytest.fixture
+def selenium(selenium):
+    """Override pytest-selenium default by changing the browser window size."""
+    selenium.set_window_size(1200, 800)
+    return selenium
+
+
 @pytest.fixture
 def browser(selenium):
     """Widgetastic browser fixture."""
     selenium.get(get_qpc_url())
     return Browser(selenium)
+
+
+@pytest.fixture
+def chrome_options(chrome_options):
+    """Set options if using a chrome webdriver.
+
+    By default testing is done in headless mode.
+    """
+    # chrome_options.add_argument('--headless')
```

Comment:
I think this line can be removed, right?

Reply:
Actually, I'm not sure here. I think in the final merge we should default to headless mode -- less memory overhead and the exact same operation.